solr/solrj/build.gradle

```diff
@@ -47,7 +45,6 @@ dependencies {
   testImplementation project(':solr:test-framework')
```

Comment:
After line 45 we need a new line for 9.x `runtimeOnly project(':solr:solrj-streaming')`

Reply:
fixed in 805e95414c20ad027b85a164207937631c975ef2